tests/core/tests/test_tmp_storages.py

```diff
@@ -85,17 +85,24 @@ def test_cache_storage(self):
     def test_cache_storage_read_with_encoding(self):
         tmp_storage = CacheStorage()
         tmp_storage.name = 'f'
-        cache.set("django-import-export-f", 101)
+        tmp_storage.save(101)
         res = tmp_storage.read(encoding='utf-8')
         self.assertEqual(101, res)
 
+    def test_cache_storage_read_with_encoding_unicode_chars(self):
+        tmp_storage = CacheStorage()
+        tmp_storage.name = 'f'
+        tmp_storage.save("àèìòùçñ")
+        res = tmp_storage.read(encoding='utf-8')
+        self.assertEqual("àèìòùçñ", res)
+
     def test_media_storage(self):
         tmp_storage = MediaStorage()
         tmp_storage.save(self.test_string)
         name = tmp_storage.name
 
         tmp_storage = MediaStorage(name=name)
-        self.assertEqual(self.test_string, tmp_storage.read())
+        self.assertEqual(self.test_string.decode(), tmp_storage.read(encoding='utf-8'))
```

Comment:
@matthewhegarty note that I needed to add `decode()` here after moving `force_str` to the tmp_storage classes. Every other test is using decode(), this could be a good reason to move decoding to tmp_storage classes, to always have the same interface.